src/components/enterprise-page/EnterprisePage.jsx

```diff
@@ -43,7 +43,7 @@ export default function EnterprisePage({ children }) {
 
   // We explicitly set enterpriseConfig to null if there is no configuration, the learner portal is
   // not enabled, or there was an error fetching the configuration. In all these cases, we want to 404.
-  if (isDefinedAndNull(enterpriseConfig)) {
+  if (isDefined(enterpriseConfig) && !enterpriseConfig) {
```

Comment:
`isDefinedAndNull(enterpriseConfig)` and `isDefined(enterpriseConfig) && !enterpriseConfig` are largely the same since `enterpriseConfig` is never anything other than `null` if falsely.

`isDefinedAndNull(enterpriseConfig)` is a more readable, IMO.